geoportal/c2cgeoportal_geoportal/views/dynamic.py

```diff
@@ -62,11 +62,14 @@ def _fulltextsearch_groups(self):  # pylint: disable=no-self-use
             .all()
         ]
 
-    def _interface(self, interface_config, interface_name, dynamic):
+    def _interface(self, interface_config, interface_name, route_interface_name, dynamic):
```

Comment:
route_interface_name is not clear for me.
root_interface_name ? or route_name ?

Reply:
=> original_interface_name with documentation :-)